omnigibson/object_states/toggle.py

```diff
@@ -51,16 +52,21 @@ def _initialize(self):
         pre_existing_mesh = get_prim_at_path(mesh_prim_path)
         # Create a primitive mesh if it doesn't already exist
         if not pre_existing_mesh:
-            button = UsdGeom.Sphere.Define(og.sim.stage, Sdf.Path(mesh_prim_path))
             self.radius = m.DEFAULT_RADIUS if self.radius is None else self.radius
+            mesh = create_primitive_mesh(
```

Comment:
Maybe add a comment here explaining we need to create a sphere mesh rather than a sphere shape because of the physx warning?

Reply:
done!